ppbkc_generateBaseData.m

```diff
@@ -164,8 +168,16 @@
     machine.data.kernel(i).kernel1 = fGradFitPar_1' ./ kernelNorm;
     machine.data.kernel(i).kernel2 = fGradFitPar_2' ./ kernelNorm;
     machine.data.kernel(i).kernel3 = fGradFitPar_3' ./ kernelNorm;
-
+    
+    if machine.data.iNumKernel == 4
+        W_ri(:,4) = tpr(1,:)./tpr(1,1);
+        D_4_spline = interp1(tprFieldSizes,W_ri(:,4),equivalentFieldSizes, 'spline');
+        fGradFitPar_4 = [correctedOutputFactorAtEquiFieldSizes(1)*D_4_spline(1), ...
+                         diff(correctedOutputFactorAtEquiFieldSizes.*D_4_spline)];
+        machine.data.kernel(i).kernel4 = fGradFitPar_4' ./ kernelNorm;        
+    end
```

Comment:
We could certainly do it that way but IMO this might be better solved by putting everthing in cell arrays with the index corresponding to the current kernel number. That way we could maybe also realize a better structure in the dose calculation within matRad, meaning avoiding the if/else statement. Could you also submit a pull request for the matRad dose calculation so we can evaluate this together?

Reply:
I will try to submit a pull request with my edited matRad version. 
I tried to merge the 'dev' with 'photonKernelVal' to get to the current state and afterwards I would implement my edited files to the 'photonKernelVal' version but it seems that I havn´t the necessary permissions to push it to this dev.
Or maybe I need some additional time to get a deeper understanding of this whole concept of pushing, merging and so on.